README.md

```diff
@@ -1804,10 +1804,12 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   ```swift
   // WRONG
+  // swiftformat:disable redundantRawValues
```

Comment:
Why is this "wrong"?

Reply:
@calda Are we talking "wrong" as comment "// WRONG"?

Reply:
I'm not sure I understand why this needs both `// WRONG` _and_ `// swiftformat:disable redundantRawValues`. Doesn't the SwiftFormat directive mean this is correct (e.g. we're manually indicating that these values should be preserved because they map to an external source)?

Reply:
It was there even before I requested PR #216 and PR #215 . And, I thought it was there as it was not relying on Swift's automatic enum values. (Error could be coming from internal source.)

If you think otherwise then shouldn't line 1807 to 1812 (enum ErrorType) switched with 1841 to 1845 (enum ErrorType)?

Reply:
> I'm not sure I understand why this needs both `// WRONG` _and_ `// swiftformat:disable redundantRawValues`. Doesn't the SwiftFormat directive mean this is correct (e.g. we're manually indicating that these values should be preserved because they map to an external source)?

Oh, I thought we were agreed to state it as wrong and make exception to show example of wrongdoings.

Reply:
haha yeah I think removing the directive is fine, otherwise this may be confusing